### PR TITLE
css-brochure-form-Dylan De Vei

### PR DESCRIPTION
**Pull request by dylandevei**
Screen Shot 2022-03-17 at 9 41 16 AM

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 15px;
+  color: white;
+}
+
+.button {
+  text-align: center;
+  float: right;
+}
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: I don't think this is doing anything for you.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+#email,
+#name,
+#hiring {
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: You should stick to applying styling to classes rather than ids.

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  padding-left: 10px;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: black;
+}
+
+#engineers {
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: You should stick to applying styling to classes rather than ids.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work! A few notes for you:

**Comment by dylandevei** [comment]
Screen Shot 2022-03-17 at 12 00 33 PM
updated some stuff dont think I figured everything out. Was just pushing so I could get back to main.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude! Just throwing a review your way so I know when this is actually ready for another review 🤓

**Comment by dylandevei** [comment]
https://user-images.githubusercontent.com/99229316/159397611-6a73f56c-d952-4131-93a0-8910374a4110.mov



Here are the final changes for css-brochure-form

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**